demos/index.html

```diff
@@ -25,6 +25,12 @@ <h1>Streams Demos</h1>
 
   <dt><a href="https://fetch-progress.anthum.com/">Progress indicators with Service Workers and Fetch</a> | <a href="https://github.com/AnthumChris/fetch-progress-indicators">(Source Code)</a>
   <dd>Streams intercept Service Worker <code>FetchEvent</code> and <code>fetch()</code> statements to show progress indicators. See <a href="https://github.com/AnthumChris/fetch-progress-indicators#browser-support">browser support</a> notes.
+
+  <dt><a href="http://mdn.github.io/dom-examples/streams/grayscale-png/">Grayscale a PNG</a> | <a href="https://github.com/mdn/dom-examples/tree/master/streams/grayscale-png">(Source Code)</a>
+  <dd>A PNG gets fetched to receive a ReadableStream on it, than the PNG data is converted to black and white and enqueued into another ReadableStream.
```

Comment:
Could you update this comment to indicate that it uses TransformStream?

Also, in the demo itself, it would nice if it displayed a message in the page when TransformStream is not defined, to avoid confusion.

Reply:
We don't need to block this on the changing http://mdn.github.io/dom-examples/streams/grayscale-png/. The MDN people might not agree to that anyway.